.github/workflows/scheduled.yaml

```
@@ -48,19 +48,20 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/branch-cleanup.yaml@main
     secrets:
       REPO_GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
-  revdepcheck:
-    if: >
-      github.event_name == 'schedule' || (
-        github.event_name == 'workflow_dispatch' &&
-        inputs.chosen-workflow == 'revdepcheck'
-      )
-    name: revdepcheck ↩️
-    uses: insightsengineering/r.pkg.template/.github/workflows/revdepcheck.yaml@main
+  # disable until the first release
+  # revdepcheck:
+  #   if: >
+  #     github.event_name == 'schedule' || (
+  #       github.event_name == 'workflow_dispatch' &&
+  #       inputs.chosen-workflow == 'revdepcheck'
+  #     )
+  #   name: revdepcheck ↩️
+  #   uses: insightsengineering/r.pkg.template/.github/workflows/revdepcheck.yaml@main
   rhub:
     if: >
       github.event_name == 'schedule' || (
         github.event_name == 'workflow_dispatch' &&
         inputs.chosen-workflow == 'rhub'
       )
     name: R-hub 🌐
-    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@main
+    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@rhub_skip_quarto
```

revert after merge of https://github.com/insightsengineering/r.pkg.template/pull/258

```suggestion
    uses: insightsengineering/r.pkg.template/.github/workflows/rhub.yaml@main
```